setup.cfg

```diff
@@ -7,6 +7,7 @@ ignore = E309
 exclude = migrations
 max-line-length = 120
 max-complexity = 10
+ignore = E203,W503
```

Comment:
is flake8 having false positives / disagreeing with black?

Reply:
Yup, there's some disagreement between both of them
https://github.com/psf/black/issues/280